x-pack/plugins/timelines/public/components/t_grid/body/index.tsx

```diff
@@ -537,10 +541,10 @@ export const BodyComponent = React.memo<StatefulBodyProps>(
           ecsData: data[rowIndex].ecs,
           browserFields,
           rowRenderers,
-        });
-      },
-      [columnHeaders, data, id, renderCellValue, tabType, theme, browserFields, rowRenderers]
-    );
+        }) as React.ReactElement;
```

Comment:
I think that this type coercion is misleading. But I don't know a good way of converting a ReacElement to a ReactNode. Ideally, `EuiDataGrid` `renderCellValue` prop should have the same type as t-grid `renderCellValue`. But that would be a massive change for this PR.